guides/plugins/apps/starter/starter-admin-extension.md

```diff
@@ -0,0 +1,149 @@
+# App-Starter - Create an Admin Extension
+
+In this guide, you will learn how to create an Admin Extension using the Admin Extension SDK.
+
+## Prerequisites
+
+> Required: Feature Flag FEATURE_NEXT_17950=1 
+
+ * Basic CLI usage (creating files, directories, running commands)
+ * We will use the following libraries / softwares
+    * npm
+    * ngrok
+    * live-server (small local development live-reloading server)
+
+## Create the App Wrapper
+
+First of all we need to create the app "wrapper", the so-called app manifest. It contains just a single file with some basic configuration.
+
+### Create manifest file
+
+We create this file in a directory where all other app resources will reside in. Let's name this `ListingExtension`. Please execute the command below in your Shopware root directory.
+
+```bash
+mkdir -p custom/apps/ListingExtension
+cd custom/apps/ListingExtension
+touch manifest.xml
+```
+
+Next, we're gonna put our basic configuration into the file we just created.
+
+{% code title="manifest.xml" %}
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+        <name>ListingExtension</name>
+        <label>Listing Extension App</label>
+        <description>This app extends the product listing</description>
+        <author>shopware AG</author>
+        <copyright>(c) shopware AG</copyright>
+        <version>1.0.0</version>
+        <license>MIT</license>
+    </meta>
+</manifest>
+
+```
+{% endcode %}
+
+## Set up communication between Shopware and the App
+
+Next, we need to set up an entry point, so Shopware and your app can communicate. The entry point is a static `.html` file, which includes the Extension SDK script and defines our extension.
+
+
+> Create a graphic illustrating communication between Shopware and the App.
+
+Let's create this file in a directory called `src`.
+
+```bash
+cd custom/apps/ListingExtension
+mkdir src
+touch src/index.html
+```
+
+```html
+<!doctype html>
+<html>
+    <head>
+        <script src="https://unpkg.com/@shopware-ag/admin-extension-sdk/cdn"></script>
+    </head>
+    <script>
+        sw.notification.dispatch({
+            title: 'Hi there',
+            message: 'Looks like someone sent you a message'
+        })
+    </script>
+</html>
+
+```
+
+### Start the local development server
+
+Next, we need to start the live server so you don't always have to reload the page manually.
+
+```bash
+cd custom/apps/ListingExtension
+npm install -g live-server
```

Comment:
You can directly use the live-server no need for ngrok.

Reply:
Is it also applicable for App development with cloud-stores or will we run into CORS issues then?